### DISCUSSION TITLE
Headless issues with Safari and Firefox

### DISCUSSION THREAD

**Discussion by minemindmedia** [alpinejs/alpine#3593] [7. Bugs (archived)] [answered]
I have been working on a project for a while and have just recently noticed that the headless modals / slideout components I am using work great in Chrome but do not work in Safari or Firefox. Headless modals open up the overlay but nothing else. The screen becomes locked and a refresh is required.

**Comment by ekwoka** [comment]
Does this happen on the example pages as well?

Or in your code only?

**Comment by minemindmedia** [comment]
I apologize about not checking that but yeah, it works fine with the example pages. I will see what I can do to debug.

**Comment by minemindmedia** [comment]
In an attempt to narrow this down, I've removed all other scripts and dependencies and after doing so, I was still experiencing the same issues with Firefox and Safari.

So, I took this very basic example and replaced my slideout with it as a test and noticed that the functionality is working in all 3 browsers.

<img width="842" alt="image" src="https://github.com/alpinejs/alpine/assets/8854361/253e19a2-afbd-4580-920e-39d1ae990f5f">

<img width="531" alt="image" src="https://github.com/alpinejs/alpine/assets/8854361/82a7620c-2c75-4a71-91d4-1fe7ff50f71c">

But as soon as I added a class, "bg-black" to the panel, it stops functioning in Safari and Firefox but works in Chrome.
<img width="657" alt="image" src="https://github.com/alpinejs/alpine/assets/8854361/446c6c8c-d9b4-42c2-aad5-ea3deb7c2f33">

If I attempt to open the modal in FF or Safari, it locks the screen (because the modal is working but not visually).
I put all my original slideout code back in place, removed all the classes from the html and now I can visually see the functionality of the slideout but stying is removed of course.

Hmm, a tailwindcss / alpinejs conflict?

**Reply by ekwoka** [reply]
That's quite strange. I'll check it out

**Comment by minemindmedia** [comment, marked as answer]
It turns out I still had the Alpine module installed via npm and that was the conflict. Still, very strange behavior with the whole class names thing but at least I determined the problem.
